### ISSUE TITLE
Blazor WASM project does not build

### ISSUE THREAD

**Issue by Akinzekeel** [koenvzeijl/AspNetCore.SassCompiler#51]
When using the `AspNetCore.SassCompiler` package in a Blazor WebAssembly project, the build process fails with the following error:

`error NETSDK1082: Für "Microsoft.AspNetCore.App" stand für den angegebenen RuntimeIdentifier "browser-wasm" kein Runtimepaket zur Verfügung.`

**To Reproduce**
1. dotnet new blazorwasm
2. dotnet add package AspNetCore.SassCompiler
3. dotnet build

**Expected behavior**
Build completes successfully

**Additional context**
I am using .Net 6.

**Comment by aDisplayName** [comment]
Same here. Also, @Akinzekeel , high five for working in new year!

**Comment by sleeuwen** [comment]
Hi, this project currently doesn't support blazor wasm. The reason for this is that we use an IHostedService to implement the watch functionality.
I currently don't know of a way to add that for a blazor wasm project. Technically we could package only the compiler and msbuild task so you can use it without a watcher, but that would require a rebuild every time you change the sass/scss files which is suboptimal (this could be useful for CI/CD though).
I don't have a lot of experience with blazor wasm so it could also be that there is a way to do this that I don't know about.

**Comment by sygmond** [comment]
@Akinzekeel check out my forked version [here](https://github.com/sygmond/AspNetCore.SassCompiler). The NuGet package I named it [AspNetCore.SassCompiler.Blazor.CI](https://www.nuget.org/packages/AspNetCore.SassCompiler.Blazor.CI/). Test it out.

Short story about changes:
- autogenerated config file named `sasscompiler.json` (settings are not inside `appsettings.json` anymore)
- ability to use `sasscompiler.Development.json` (i.e. when you want expanded css in development and compressed in production)
- now the Service Watcher and the Tasks read the settings from the same config file
- added `--no-source-map` to default settings

For detailed list of changes, check the merge request, README.md or the commit message.

If the merge request is accepted, my NuGet will become obsolete/old and I recommend using the original NuGet as it will be maintained more frequently.
I just created the NuGet packege to be able to use my changes in Docker, as I didn't have time to wait for the merge request approval.
